spec/google/gax/api_callable_spec.rb

```diff
@@ -211,8 +211,8 @@ def initialize(msg, code)
       call_count = 0
 
       time_now = Time.now
-      allow(Time).to receive(:now).exactly(4).times.and_return(
-        *([time_now] * to_attempt + [time_now + 2])
+      allow(Time).to receive(:now).exactly(7).times.and_return(
```

Comment:
This code mocks Time.now, and the number of invocation changes because Time.now can be called twice per API call (one in retryable and the other, the new one, in add_timeout_arg).

Added comments to describe, also replaced the number (7) by the calculation from to_attempt.
PTAL.